src/components/Routing.js

```diff
@@ -273,6 +274,11 @@ class Routing extends Component {
                                 path='/recover-with-link/:accountId?/:seedPhrase?'
                                 component={RecoverWithLinkWithRouter}
                             />
+                            <Route
+                                exact
+                                path='/auto-recover'
```

Comment:
@Patrick1904 @MaximusHaximus I agree with this concern, however I think to be consistent with other routes it would be smth like

`/auto-recover-seed-phrase#account-id.near/<seed phrase>`

I think "auto" is happening here, as there is no confirmation involved when importing.

Also I wonder if user actually has control over this seed phrase. We should in Profile that seed phrase is enabled, but did user actually at any point write it down?

@mattlockyer @corwinharrell maybe you have better context on this?

Reply:
Not sure I quite understand what the question is here, but here's some context on the recover vs import language:

We were initially using titles and labels in the UI that said: "Recover". While this is common, we felt that it didn't reflect the needs of users who were simply looking to add an existing account to their current session. "Recover" implies that something has been lost which is not always the case, and we felt that "Import" did a better job of reflecting the action of adding an account (perhaps on a new machine, incognito window, etc...)

While the titles and button labels were changed to reflect this, I don't know if the actual routes were. I'm still in favor of the "Import" language for our UI and routes where the user is looking to add an additional account to their growing list.

Reply:
If we want to keep route consistency, take into consideration the fact that the account is automatically imported, and 'import' language, I suggest we do something like this:

`/auto-import-seed-phrase#johndoe.near/decline agree blue sentence pumpkin use venture delay crisp retreat cake hover`

> Also I wonder if user actually has control over this seed phrase. We should in Profile that seed phrase is enabled, but did user actually at any point write it down?

Good point. If my account was created outside of the NEAR Wallet, I'd prefer if the seed phrase was shown as _not enabled_ on the profile, so that I can set one up if I want to. We may also want to add some type of banner on the profile saying 'Please enable a Wallet recovery method', etc.